src/Identity.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Authentication;
+
+use Cake\Core\InstanceConfigTrait;
+
+/**
+ * Identity object
+ */
+class Identity implements IdentityInterface
```

Comment:
Strange, I thought I did this, I wrote it down. Not sure why it's missing. Going to add it back.